source/MaterialXCore/Observer.h

```diff
@@ -61,7 +61,8 @@ class ObservedDocument : public Document
   public:
     ObservedDocument(ElementPtr parent, const string& name) : 
         Document(parent, name),
-        _updateScope(0)
+        _updateScope(0),
+        _notificationsEnabled(true)
```

Comment:
Following up on the suggestion above, I would recommend "_callbacksSuspended" for the name of this variable.